packages/react-widgets/src/models/fqn.js

```diff
@@ -26,13 +26,15 @@ const fqnParseRegex = {
     `^${databricksIdentifierRegex}(?:\\.${databricksIdentifierRegex})?(?:\\.${databricksIdentifierRegex})?$`
   )
 };
+fqnParseRegex[Provider.DatabricksRest] = fqnParseRegex[Provider.Databricks];
```

Comment:
_Suggestion:_ I'll add this for consistency inside of `fqnParseRegex` and will extract the Regex, rather than updating the object afterward, increasing (at least, IMHO) this part's readability, e.g.

```
const databricksRegex = `^${databricksIdentifierRegex}(?:\\.${databricksIdentifierRegex})?(?:\\.${databricksIdentifierRegex})?$`
const fqnParseRegex = {
  ...
  [Provider.Databricks]: databricksRegex,
  [Provider.DatabricksRest]: databricksRegex
}
```

Same applies to `nameNeedsQuotesChecker`

Reply:
Applied more or less.